layout/_third-party/comments/utteranc.swig

```diff
@@ -0,0 +1,9 @@
+{% set utteranc_js_url = '//utteranc.es/client.js' %}
+
+<script src="https:{{utteranc_js_url}}"
+        repo="{{ theme.utteranc.repo }}"
+        issue-term="{{ theme.utteranc.pathname }}"
+        theme="{{ theme.utteranc.theme }}"
+        crossorigin="anonymous"
+        async>
+</script>
```

Comment:
```
<script src="https:{{utteranc_js_url}}"
  repo="{{ theme.utteranc.repo }}"
  ...
  async>
</script>

Reply:
We use 2 spaces indentation 🚀

Reply:
I think no need `https` here. This not google, right?